src/components/AskFieldWrapper.js

```diff
@@ -43,7 +46,12 @@ class AskFieldWrapper extends Component {
     this._field = component
   }
 
+  setValidationMessage (message) {
+    this.setState({ validationMessage: message })
+  }
+
   render () {
+    console.log(this.props)
     var widgetSpec = this.props
```

Comment:
render accepts 2 parameters, props and state, usually we should do

```
render(props, state) {
}
```
depending if you need/want to use state and/or props